### ISSUE TITLE
Use spectrogram for caculate assessment

### ISSUE THREAD

**Issue by phanngoc** [phanngoc/english-pronunciation#4]
def get_embedding(audio_path):
    audio_input = tokenizer(audio_path, return_tensors="pt").input_values
    with torch.no_grad():
        embedding = model(audio_input).last_hidden_state.mean(dim=1)
    return embedding.numpy().flatten()

user_embedding = get_embedding(user_path)
standard_embedding = get_embedding(standard_path)

# Compare embeddings
similarity = 1 - cosine(user_embedding, standard_embedding)
print(f"Embedding Cosine Similarity: {similarity}")
